src/App.js

```diff
@@ -1,24 +1,142 @@
-import activities from './resources/schedule.json'
-import dayjs from "dayjs";
+import { useEffect, useState } from 'react';
+import activities from './resources/schedule.json';
+import dayjs from 'dayjs';
 
 function App() {
+  const [today, setToday] = useState([]);
+  const [tomorrow, setTomorrow] = useState([]);
+  const [nextWeek, setNextWeek] = useState([]);
+  const [future, setFuture] = useState([]);
+
+  const todayDate = new Date();
+  let tomorrowDate = new Date();
+  tomorrowDate.setDate(todayDate.getDate() + 1);
+  const thisWeekEnds = new Date();
+  thisWeekEnds.setDate(todayDate.getDate() + 2);
```

Comment:
You're solving the work relative to the day you're doing it. What if it was a Monday, will you update your code to +5? This doesn't work in the real world